hw/ip/aes/doc/_index.md

```diff
@@ -74,7 +74,7 @@ The AES unit automatically starts the encryption/decryption of the next data blo
 The order in which the input registers are written does not matter.
 Every input register must be written at least once for the AES unit to automatically start encryption/decryption.
```

Comment:
Do you mean input data register? 
if not, it this also true when running ECB mode - it does not make sense if the user have to write the IV registers for ECB

Reply:
Yes, this section is about the input data registers.